### DISCUSSION TITLE
More transparency options for scenery

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#16780] [Feature requests]
Water's transparency seems a little different than scenery - not only does it change the palette below to water colors, but it can also affect the shade. It would be nice if scenery objects could be made to exactly match terrain water in transparency, that way parks using custom water can still have the water look the same as regular water.

**Comment by 733737** [comment]
on a separate note, could you remove glitches from the water transparency when using tile inspector to change the scenery render order?

**Reply by spacek531** [reply]
That is probably outside the scope of this feature.

**Comment by 733737** [comment]
ok, well sorry, just thought it was relevant to mention how glitchy that might be to implement

**Comment by 733737** [comment]
by the way, water rides only use sparkle colors, not the water colors, did you know that?

**Reply by spacek531** [reply]
That's an intriguing opinion but it is incorrect.

**Reply by 733737** [reply]
how is it intriguing if you are so sure it is untrue?

**Comment by 733737** [comment]
actually you are right, sorry
![sample](https://user-images.githubusercontent.com/94884086/158039418-5ed99801-7f19-4323-8be6-6111c53fdf22.png)
